python/src/com/jetbrains/python/findUsages/PyClassFindUsagesHandler.java

```diff
@@ -28,7 +27,7 @@
 /**
  * @author yole
  */
-public class PyClassFindUsagesHandler extends FindUsagesHandler {
+public class PyClassFindUsagesHandler extends PyFindUsagesHandler {
```

Comment:
Let's add our Python-specific handler as the base class for all Python find usages handlers, including `PyModuleFindUsagesHandler`.